### DISCUSSION TITLE
How to remove outline of checkbox on focus.

### DISCUSSION THREAD

**Discussion by yuta-hidaka** [tailwindlabs/tailwindcss#9597] [Help] [answered]
Hi all !!

I'm using https://github.com/tailwindlabs/tailwindcss-forms on our project and applying design to forms.
But I have faced issue that is checkbox appire some outline when focused.

So I'm tring to remove this checkbox's outline but outline-none dose not work for this.

So could you tell me the way to remove this outline ? 

You can see blue outline in this picture on focus.
<img width="364" alt="image" src="https://user-images.githubusercontent.com/46583585/196360210-a9009af8-6943-4fa8-a49d-74949aaf11b5.png">

## minimal example
https://play.tailwindcss.com/ILz1a1GBYK

## code
```html
<div class="m-5">
  <input class="form-checkbox focus:outline-none focus-visible:outline-none" type="checkbox" />
</div>

```

**Comment by wongjn** [comment, marked as answer]
In `@tailwindcss/forms`, [the outline is constructed using the `ring-` utilities](https://github.com/tailwindlabs/tailwindcss-forms/blob/b3ed1576aa886bdca96b6d7c6d8ab3bf7ea5e7d5/src/index.js#L48-L54). So to remove it, you could try using `focus:ring-0` or `focus:ring-transparent`.

**Reply by Pariola-droid** [reply]
Thanks a bunch!

**Comment by yuta-hidaka** [comment]
@wongjn 

You are save my day!!!!

Thank you so much quick response!!!!

**Comment by owen2jan** [comment]
also you can add `focus:ring-offset-0`

**Reply by picardplaisimond** [reply]
You saved me a ton of time ! Thank you !

**Reply by devantic** [reply]
TY! This was the final piece of the puzzle to get a completely naked check mark. This does it:

```bg-transparent border-none text-transparent focus:ring-0 focus:ring-offset-0```

**Reply by ZackPlauche** [reply]
Life saver haha.

**Reply by SchunckLeonardo** [reply]
Omg! Thank You!

**Reply by raspacee** [reply]
Has this stopped working? It does not work anymore

**Comment by paciodev** [comment]
As for now you can remove it by using:
`focus-visible:outline-none`

**Reply by aditya-1310** [reply]
Thanks, you are time saver!
